packages/@ourworldindata/grapher/src/mapCharts/ChoroplethGlobe.tsx

```diff
@@ -0,0 +1,711 @@
+                    // ignore the delta-0 events. This fixes a bug where the
+                    // rendered SVG country paths would interfere with the
+                    // pinch-to-zoom gesture.
+                    if (delta === 0) return
+
+                    // Rotate to the midpoint of the pinch gesture while zooming
+                    const midPoint = calculatePinchMidPoint(event.sourceEvent)
+                    this.rotateAndZoomGlobe(midPoint, delta)
+
+                    startDistance = distance
+                }
+
+                const panning = (): void => {
+                    // True if the cursor is currently over the globe
+                    const isDraggingGlobe = isPointInCircle(posVector, {
+                        cx: this.globeCenter[0],
+                        cy: this.globeCenter[1],
+                        r: this.globeRadius,
+                    })
+
+                    if (isDraggingGlobe) {
+                        const targetRotation = calculateTargetRotation()
+                        this.rotateGlobe(targetRotation)
+                    } else {
+                        // If the user's cursor is outside of the globe, then
+                        // adjust the globe's rotation based on the cursor's
+                        // movement, applying a sensitivity factor to control
+                        // the speed of rotation
+
+                        const sensitivity = 0.8
+                        const r = this.globeRotation
+                        const dx = pos[0] - previousPos[0]
+                        const dy = pos[1] - previousPos[1]
+                        this.rotateGlobe([
+                            r[0] + sensitivity * dx,
+                            r[1] - sensitivity * dy,
+                        ])
+                    }
+
+                    previousPos = pos
+                }
+
+                const type = getInteractionType(event)
+
+                // bail if a zoom-pinch gesture turned into a pan
+                // because this might lead to erratic jumps
+                if (previousType === "zoom-pinch" && type === "pan") return
+
+                if (type === "zoom-scroll") wheeling()
+                else if (type === "zoom-pinch") pinching()
+                else if (type === "pan") panning()
+
+                previousType = type
+            }
+
+            const end = (): void => {
+                this.isPanningOrZooming = false
+                startDistance = undefined
+                previousType = undefined
+            }
+
+            return zoom()
+                .scaleExtent([MIN_ZOOM_SCALE, MAX_ZOOM_SCALE])
+                .touchable(() => this.isTouchDevice)
+                .on("start", start)
+                .on("zoom", panningOrZooming)
+                .on("end", end)
+        }
+
+        select(base).call(panAndZoom())
+    }
+
+    componentDidMount(): void {
+        document.addEventListener("touchstart", this.onDocumentClick, true)
```

Comment:
We should make this into a passive event listener, so it doesn't interfere with scrolling or any other touch interactions:
```suggestion
        document.addEventListener("touchstart", this.onDocumentClick, { capture: true, passive: true })
```